test/programs/pizza-ordering/cli/entry.js

```diff
@@ -1,4 +1,5 @@
 // Initialize Waterfall CLI
-require('../../../../dist/index.js')({
+const cli = require('../../../../dist');
+cli.default({
```

Comment:
The approach here should not change. It should simply add `.default` after the require, like:

```
require('../../../../dist').default({
```

This suggestion also removes `index.js`.